src/main/java/com/undertheriver/sgsg/foler/service/FolderService.java

```diff
@@ -48,6 +49,11 @@ private boolean isFoldersExistsMoreThan20(Long userId, Integer limit) {
 			.collect(Collectors.toList());
 	}
 
+	@Transactional(readOnly = true)
+	public Folder read(Long folderId) {
+		return folderRepository.findById(folderId).get();
+	}
```

Comment:
.get()으로 그대로 꺼내면 JPA에서 지정해주는 Exception이 나올텐데요.
제가 정의해놓은 ModelNotFoundException을 이용해주세요! 나중에 예외핸들링 하기 좋을 것 같네요.

```suggestion
	public Folder read(Long folderId) {
		return folderRepository.findById(folderId)
		  .orElseThorow(() -> new ModelNotFoundException...);
	}
```

Reply:
앗 맞다.. 정의해주신걸로 변경했습니다!